index.html

```diff
@@ -0,0 +1,460 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <script src="https://kit.fontawesome.com/ea6955352c.js" crossorigin="anonymous"></script>
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <button type="button" class="menu-bars">
+        <i class="fas fa-bars"></i>
+      </button>
+      <nav id="mobile-menu" class="not-active">
+        <div>
+          <i id="close" class="fas fa-xmark"></i>
+          <ul>
+            <li><a class="mobile-link" href="#projects>Protfoliu</a></li>
+            <li><a class="mobile-link" href="#about-me">About</a></li>
+            <li>
+              <a class="mobile-link" href="#contact">Contact</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+      <nav id="desktop-nav>
+        <ul>
+          <li><a class="nav-link" href="#projects">Portoflio</a></li>
+          <li><a class="nav-link" href="#about-me">About</a></li>
+          <li>
+            <a class="nav-link" href="#contact"
+              >Contact <i class="far fa-envelope"></i
+            ></a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div id="intro">
+        <div class="intro-icon">
+          <img
+            class="mobile-icon"
+            src="./media/svg/icon.svg"
+            alt="Icon for the side"
+          />
+          <img
+            class="desktop-icon"
+            src="./media/svg/desktop-icon.svg"
+            alt="Icon for the side"
+          />
+        </div>
+        <h2 class="intro-greet">
+          Hey There.
+          <br />
+          I'm Aziz
+        </h2>
```

Comment:
I suggest using h1 instead of h2 for the header to follow the HTML best practices. don't hesitate to contact me if any help is needed!